client/order/index.js

```diff
@@ -243,11 +256,32 @@ const DisputeNotice = ( { chargeId } ) => {
 						);
 					},
 				},
-			] }
+			];
+
+			warningText = `${ title } ${ suffix }`;
+		}
+	}
+
+	if ( ! showWarning && ! disableRefund ) {
+		return null;
+	}
+
+	return (
+		<BannerNotice
+			status={ urgency }
+			isDismissible={ false }
+			actions={ actions }
 		>
-			<strong>
-				{ title } { suffix }
-			</strong>
+			{ showWarning && <strong>{ warningText }</strong> }
+
+			{ disableRefund && (
+				<div>
+					{ __(
+						'Refunding and order editing are disabled during disputes to maintain record integrity.',
```

Comment:
Putting my merchant hat on, 'maintain record integrity' wouldn't mean anything to me.

```suggestion
						'Refund and order editing options are unavailable at the moment because this order is in dispute.',
```